slice_conversion.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2013-2018 by Maxim Bublis <b@codemonkey.ru>
```

Comment:
This is a brand new file, with no ties to the old satori UUID package. What copyright notice is appropriate? I copied and pasted it from another file, but I don't think it's what I should be using.